app/services/start_new_week.rb

```diff
@@ -0,0 +1,12 @@
+class StartNewWeek
+  def self.call
+    active_users_ids = UserAvailability.all.pluck(:user_id).uniq
+    User.where.not(id: active_users_ids).update_all(active: false)
+    UserAvailability.destroy_all
```

Comment:
We want to keep UA which is recurring: true, right?

Reply:
that's handled in a different PR